tests/e2e.sh

```diff
@@ -100,3 +99,47 @@ docker run -u "$(id -u)" -v "$(pwd)/${CHARTS_SRC}":/source gcr.io/kpt-functions/
 assert_dir_exists default
 assert_contains_string default/secret_my-mongodb.yaml "my-mongodb"
 assert_contains_string default/secret_my-redis.yaml "my-redis"
+
+############################
+# kpt fn Tests
+############################
+
+testcase "kpt_kubeval_success"
+kpt pkg get https://github.com/instrumenta/kubeval.git/fixtures .
+cat >fc.yaml <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image:  gcr.io/kpt-functions/kubeval:dev
+        network:
+          required: true
+    config.kubernetes.io/local-config: 'true'
+EOF
+kpt fn source fixtures/valid* |
+kpt fn run --fn-path fc.yaml --network >out.yaml
```

Comment:
What's the exit code if there's an unexpected failure? In that case, wouldn't you exit here and never run subsequent lines?

Reply:
Yes, you error out with exit status 1 and the whole e2e test fails. See line 144 for the failure condition.